src/saturn/game.c

```diff
@@ -521,9 +521,9 @@ INCLUDE_ASM("asm/saturn/game/f_nonmat", f6079A2C, func_06079A2C);
 INCLUDE_ASM("asm/saturn/game/f_nonmat", f6079AF0, func_06079AF0);
 INCLUDE_ASM("asm/saturn/game/f_nonmat", f6079B74, func_06079B74);
 
-void func_06079B9C(s32* param_1) {
-    param_1[1] += param_1[3];
-    param_1[2] += param_1[4];
+void MoveEntity(Entity* entity) {
+    LOW(entity->posX) += entity->accelerationX;
```

Comment:
Just FYI I'm planning on doing the `f32` refactor eventually but this is fine for now.